assets/src/edit-story/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -0,0 +1,270 @@
+/*
```

Comment:
You could even break each action into it's own files like the story provider. https://github.com/google/web-stories-wp/tree/main/assets/src/edit-story/app/story/useStoryReducer/reducers

Reply:
IMO here it's still manageable to have all in one file. Can be reconsidered once the list of reducers is growing.